build/bubbleWrapper.ts

```diff
@@ -118,36 +118,15 @@ export class BubbleWrapper {
             password: signingInfo.storePassword,
             keypassword: signingInfo.keyPassword,
             alias: signingInfo.alias,
-            fullName: this.getEscapedSigningKeyText(signingInfo.fullName),
-            organization: this.getEscapedSigningKeyText(signingInfo.organization),
-            organizationalUnit: this.getEscapedSigningKeyText(signingInfo.organizationalUnit),
-            country: this.getEscapedSigningKeyText(signingInfo.countryCode)
+            fullName: signingInfo.fullName,
+            organization: signingInfo.organization,
+            organizationalUnit: signingInfo.organizationalUnit,
+            country: signingInfo.countryCode
         };
         
         await keyTool.createSigningKey(keyOptions, overwriteExisting);
     }
 
-    private getEscapedSigningKeyText(input: string): string {
```

Comment:
Removed because Bubblewrap 1.8.2 does this automatically now.

Reply:
woot woot